Source/Charts/Renderers/LineChartRenderer.swift

```diff
@@ -785,6 +796,11 @@ open class LineChartRenderer: LineRadarRenderer
 
         // create a new path
         let to = Int(ceil(CGFloat(to - from) * phaseX + CGFloat(from)))
+
+        guard (from + 1) < to else {
```

Comment:
Why do we need this?
Maybe `let from = from + 1` instead?
Would be nice to have a comment for why we use `+ 1`

Reply:
> Why do we need this?

this's because path generations starts from command `move` for the first point, and then `add line`.